src/IK/include/BiomechanicalAnalysis/IK/InverseKinematics.h

```diff
@@ -416,19 +418,19 @@ class HumanIK
     bool updateFloorContactTasks(const std::unordered_map<int, Eigen::Matrix<double, 6, 1>>& wrenchMap);
 
     /**
-     * set the calibration matrix between the IMU and the link
-     * @param node node number
-     * @param I_R_IMU calibration matrix
+     * remove the offset on the yaw of the IMUs world
+     * @param nodeStruct unordered map containing the node number and the IMUs measurements
```

Comment:
Is the expected structure of this structure (i.e. what are the keys) described somewhere? If yes, it would be useful to reference to that here.

Reply:
done in https://github.com/ami-iit/biomechanical-analysis-framework/pull/45/commits/d989bc71410a7b49eb2d2db0afe727960cb77973